cadquery/cq.py

```diff
@@ -2392,7 +2393,7 @@ def each(
         self: T,
         callback: Callable[[CQObject], Shape],
         useLocalCoordinates: bool = False,
-        combine: Union[bool, str] = True,
+        combine: CombineMode = True,
```

Comment:
I already discussed my opinion on adding `combine` arg for this method so I won't talk about it again but in any case if we decide to add it we should probably write somewhere that the new default behaviour is to `combine` with the base which wasnt the case before. Same remark goes for `eachpoint`

Reply:
Good point. Do you think it would make sense to set it to False? I'm kind of hesitating about it.

Reply:
Setting it to False would make things transparent but we would loose consistency with other methods. 
I don't think each and eachpoint are that used so a breaking change here would probably not be a big deal.

Reply:
OK, I added a note in changes.md